example/win-how-to-build.md

```diff
@@ -0,0 +1,27 @@
+# Notes for building Windows example
+
+# Install packages
+
+```
+yarn install
+cd example
+yarn install
+```
+
+# Copy react-native-device-info module from parent folder to example folder
+
+```
+cd ..
+mkdir -p example/node_modules/react-native-device-info
+dev-sync
+cp package.json example/node_modules/react-native-device-info
```

Comment:
is that `cp package.json ...` necessary? I never need to do that for ios/android?

Reply:
seems to be not needed after i added "react-native-device-info" into package.json